woocommerce/woocommerce
# Fix some focus issues in selectWoo

Fixes #17744
Fixes #17497
Closes https://github.com/woocommerce/selectWoo/issues/13

To test fix for #17744:
- Go to the Checkout page
- Open the Country dropdown. With the country dropdown open, click on the State dropdown to open it. Verify focus isn't on the first dropdown any more and typing doesn't open the first dropdown.
- Open the country dropdown. With the country dropdown open, click on a text input field (e.g. Company Name). Verify focus moves to the clicked-on text input. 
- See videos in #17744 if these instructions are unclear.

To test fix for #17497:
- Follow steps to reproduce in #17497 and verify problem is fixed.

To test selectWoo issue #13:
- Focus on a selectWoo element then open the dropdown with the down key on the keyboard.
